target/linux/ramips/image/mt7620nand.mk

```diff
@@ -8,7 +8,7 @@ define Device/miwifi-r3
   DTS := MIWIFI-R3
   BLOCKSIZE := 128k
   PAGESIZE := 2048
-  KERNEL_SIZE := 1472k
+  KERNEL_SIZE := 1440k
```

Comment:
Is this KERNEL_SIZE field based on a real limitation in the kernel partition size, or did you put in a hardcoded value based on whatever size your build produced?

Reply:
Maybe there's a bad block in nand flash? Run `dmesg | grep nand`.